components/IscnUploadForm.vue

```diff
@@ -451,6 +456,106 @@ export default class UploadForm extends Vue {
     }
   }
 
+  async processEPub({ buffer, file }: { buffer: ArrayBuffer; file: File }) {
+    try {
+      const Book = ePub(buffer)
```

Comment:
prefer lowercap naming (`book` or `bookInstance`) for object instances, capital name are for classes